src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/css/app.css

```diff
@@ -22,6 +22,28 @@ a, .btn-link {
     padding-top: 1.1rem;
 }
 
+svg {
+    width: 200px;
+    height: 200px;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+}
+
+circle {
+    fill: none;
+    stroke: #e0e0e0;
+    stroke-width: 8px;
+    transform: rotate(-90deg);
+    transform-origin: 50% 50%;
+}
```

Comment:
This'll apply to _all_ `svg` & `circle` elements, respectively. Let's make these classes instead.

Reply:
Or maybe use `#blazor-default-loading` as a prefix similar to `#blazor-error-ui`.